src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.Double.cs

```diff
@@ -86,7 +87,13 @@ private void WriteNumberValueIndented(double value)
             JsonWriterHelper.WriteIndentation(output.Slice(BytesPending), indent);
             BytesPending += indent;
 
-            bool result = Utf8Formatter.TryFormat(value, output.Slice(BytesPending), out int bytesWritten);
+#if BUILDING_INBOX_LIBRARY
+            var format = new StandardFormat('G');
+#else
+            var format = new StandardFormat('G', 17);
+#endif
```

Comment:
Nit:
```C#
var format =
#if BUILDING_INBOX_LIBRARY
    new StandardFormat('G');
#else
    new StandardFormat('G', 17);
#endif
```

Reply:
Is that the convention we use elsewhere? I find that much harder to parse/rationalize.

Reply:
We're mixed.  My personal preference is to try to minimize the differences between ifdef branches as much as possible, factoring out what's the same so that the differences are highlighted.  I almost suggested going as far as factoring it out up through the 'G', but that got a bit awkward.  To me this version strikes a nice balance, but if you're really opposed, what you have is ok.

Reply:
How do you feel about the following instead?
```cs
StandardFormat format;

#if BUILDING_INBOX_LIBRARY
    format = new StandardFormat('G');
#else
    format = new StandardFormat('G', 17);
#endif
```

My only real problem with the former is that I find mixing code in and outside an `#ifdef` is easy to break and hard to parse (and almost never formats nicely).

Reply:
Reasonable compromise.  Thanks.